spec/spec_helper.rb

```diff
@@ -62,7 +62,7 @@
   #   # Allows RSpec to persist some state between runs in order to support
   #   # the `--only-failures` and `--next-failure` CLI options. We recommend
   #   # you configure your source control system to ignore this file.
-  #   config.example_status_persistence_file_path = "spec/examples.txt"
+  config.example_status_persistence_file_path = "spec/examples.txt"
```

Comment:
Can you uncomment the connected comment and bracketing spaces, too, please?

Reply:
👍